src/auth/baseexternalclient.ts

```diff
@@ -71,6 +71,7 @@ export interface BaseExternalAccountClientOptions {
   client_id?: string;
   client_secret?: string;
   quota_project_id?: string;
+  workforce_pool_user_project?: string;
```

Comment:
Since this is specific to IdentityPoolClient, I think this should be in IdentityPoolClientOptions

Reply:
this one also @bojeil-google